src/couch/src/couch_query_servers.erl

```diff
@@ -278,7 +278,7 @@ sum_arrays(Else, _) ->
     throw_sum_error(Else).
 
 check_sum_overflow(InSize, OutSize, Sum) ->
-    Overflowed = OutSize > 4906 andalso OutSize * 2 > InSize,
+    Overflowed = OutSize > overflow_threshold() andalso OutSize * overflow_ratio() > InSize,
```

Comment:
Originally these are mirroring the ones from the custom view reducer. It would odd if we made these configurable but not those. For those it's a bit more elaborate as we'd have to pass them through to the js view engines and handle the overflow in javascript.  Do we just want to adjust the thresholds perhaps? I can see maybe 1.5 being a better value?